frontend/src/view/global-components/authenticated-route.component.tsx

```diff
@@ -8,7 +8,7 @@ export const AuthenticatedRoute: FC<{ children?: ReactNode } & any> = ({ childre
 
   useEffect(() => {
     setCanAccess(auth.isConnected());
-  }, []);
+  }, [auth]);
```

Comment:
Already told you in the previous review that this will call the useEffect constantly, so this dependency array should be empty. We only need the effect to apply at component load!